src/main/java/org/springframework/data/r2dbc/function/DatabaseClient.java

```diff
@@ -245,15 +247,15 @@ static DatabaseClient create(ConnectionFactory factory) {
 		 * Specify the target {@literal table} to insert into.
 		 *
 		 * @param table must not be {@literal null} or empty.
-		 * @return
+		 * @return a {@link GenericInsertSpec} for further configuration of the insert. Guaranteed to be not {@code null}.
 		 */
 		GenericInsertSpec<Map<String, Object>> into(String table);
 
 		/**
 		 * Specify the target table to insert to using the {@link Class entity class}.
 		 *
 		 * @param table must not be {@literal null}.
-		 * @return
+		 * @return a {@link TypedInsertSpec} for further configuration of the insert. Guaranteed to be not {@code null}.
```

Comment:
We typically use `{@literal null}` instead of `{@code null}`.

Reply:
But, but, but that is wrong. `@literal` is to avoid interpretation of HTML entities and similar. `null` is certainly a piece of code and should as such marked up with `@code`. :-(

Reply:
Indeed. Let's stick for consistency to `@literal` and discuss the issue within our team.